doctestplus failure with pytest 8.1.1

pytest 8.1.1 (just released) contains a `consider_namespace_packages` argument for `import_path`. The changes added in https://github.com/scientific-python/pytest-doctestplus/pull/241 use this argument for pytest `>=8.2`:
https://github.com/scientific-python/pytest-doctestplus/blob/9b747b879c2eeb6eabd7355154c1c1f5070e7e9f/pytest_doctestplus/plugin.py#L268-L271

but unfortunately the above argument change was included in a "bugfix" 8.1.1 version. This produces errors like the following:
```
.tox/py39-parallel-cov/lib/python3.9/site-packages/pytest_doctestplus/plugin.py:268: in collect
    module = import_path(fspath, mode=mode, root=self.config.rootpath)
E   TypeError: import_path() missing 1 required keyword-only argument: 'consider_namespace_packages'
```
when running pytest 8.1.1 with pytest-doctestplus 1.2.0

--- a/CHANGES.rst
+++ b/CHANGES.rst
@@ -2,6 +2,7 @@
 ==================
 
 - Compatibility with pytest 8.2. [#241]
+- Compatibility with pytest 8.1.1 [#242]
 
 1.2.0 (2024-03-04)
 ==================

--- a/pytest_doctestplus/plugin.py
+++ b/pytest_doctestplus/plugin.py
@@ -32,6 +32,7 @@ PYTEST_GT_5 = _pytest_version > Version('5.9.9')
 PYTEST_GE_5_4 = _pytest_version >= Version('5.4')
 PYTEST_GE_7_0 = _pytest_version >= Version('7.0')
 PYTEST_GE_8_0 = _pytest_version >= Version('8.0')
+PYTEST_GE_8_1_1 = _pytest_version >= Version('8.1.1')
 PYTEST_GE_8_2 = any([_pytest_version.is_devrelease,
                      _pytest_version.is_prerelease,
                      _pytest_version >= Version('8.2')])
@@ -265,7 +266,7 @@ def pytest_configure(config):
                     from _pytest.pathlib import import_path
                     mode = self.config.getoption("importmode")
 
-                if PYTEST_GE_8_2:
+                if PYTEST_GE_8_1_1:
                     consider_namespace_packages = self.config.getini("consider_namespace_packages")
                     module = import_path(fspath, mode=mode, root=self.config.rootpath,
                                          consider_namespace_packages=consider_namespace_packages)

